alerts/openshift-virtualization-operator/NodeStatusMaxImagesExceeded.md

```diff
@@ -0,0 +1,22 @@
+# NodeStatusMaxImagesExceeded
+
+## Meaning
+
+This alert is applicable to clusters that have a large number of container
+images per node. It is triggered when a node exceeds the maximum number of
+configured images (default: 50). Once this limit is reached, the scheduler is
+unable to schedule new pods that require new images on that node.
+
+## Impact
+
+Pod scheduling might be imbalanced across nodes.
```

Comment:
@machadovilaca you are missing the Diagnosis section. What command to run in order to see the issue.

Reply:
I don't think there is much to diagnose here. The alert will say what node has the issue and the problem is that it has too many images.
At most I can add a command to see the images in the node or the total count, maybe the current max config

Reply:
Yes, Can these help with identifying what can be detected?

Reply:
I think Diagnosis would be a short procedure:

1. Get image count for node.
2. If image count is greater than the default (50), resolve the issue. 

Assuming that the user has not changed the parameter, the max will be 50.

Reply:
updated

Reply:
LGTM